java-engine/src/main/java/io/getunleash/engine/CustomStrategiesEvaluator.java

```diff
@@ -125,25 +134,25 @@ private VersionedFeatures(@JsonProperty("features") List<FeatureDefinition> feat
     }
   }
 
-  private static class FeatureDefinition {
+  static class FeatureDefinition {
```

Comment:
I've exposed a few classes and methods as default access modifier so that we can poke them a little in tests. It isn't ideal to expose this kind of thing for testing purposes but a) it's not public b) making this testable without exposing these is a bigger more fragile change